etc/profile-m-z/mediathekview.profile

```diff
@@ -27,6 +29,7 @@ include disable-interpreters.inc
 include disable-programs.inc
 include disable-xdg.inc
 
+whitelist ${HOME}/.mediathek3
 include whitelist-var-common.inc
```

Comment:
Does it still work if we add the below to the profile? If not, leave them out. We can always try to tighten the profile in a follow-up PR.

include whitelist-common.inc
include whitelist-run-common.inc
include whitelist-runuser-common.inc
include whitelist-usr-share-common.inc

Reply:
If we start whitelisting, we should include wc.

Reply:
Goes in a new pr. For now I'll pull this in to get it out of the way.